core/src/window_service.rs

```diff
@@ -292,7 +291,6 @@ where
 }
 
 struct RepairMeta {
-    _from_addr: SocketAddr,
```

Comment:
@carllin is this needed still? seems this is added at https://github.com/solana-labs/solana/pull/10109#discussion_r1174807289

Reply:
yeah i think we can remove this